### PR TITLE
General: update readme.txt

### PR DESCRIPTION
**Pull request by robertbpugh**
#### Changes proposed in this Pull Request:

* Updated plugin description

#### Jetpack product discussion

* pbNhbs-eE-p2

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Please check for any spelling, grammar, or technical errors. 
* Double check the formatting is correct and presents correctly in context in the plugin and on WP.org.

#### Proposed changelog entry for your changes:

* N/A

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,97 +1,245 @@
...
+= INTEGRATIONS =
+Jetpack is updated monthly to ensure seamless integration with top WordPress plugins and other tech products.
+
+* Built for WooCommerce: Jetpack and WooCommerce are both made by Automattic. Backup, Scan, Anti-spam, integrate perfectly for Woo / eComm stores
+* Better understand your customers and marketing with Google Analytics (GA) integration
+* Social media platforms: Instagram, Facebook, Twitter, LinkedIn
+* Simple Blocks to customize your site: Pinterest, Whatsapp, Podcast player, GIFs, maps, tiled gallery, slideshow
+* Payment processors: easily collect payments or donations and sell products through Stripe and PayPal
+* Site speed and performance plugins: Works great with WP Super Cache and  Cloudflare.
+* Contact form: Anti-spam (Powered by Akismet) blocks spam comments for Jetpack forms and Contact Form 7.
```
Comment (jeherve) [inline readme.txt:70]: It will work for a lot of other form providers; maybe we should say something like "and more"?

File: readme.txt [hunk 2]
```diff
@@ -1,97 +1,245 @@
...
+* Google Analytics (GA) — Track your WordPress site statistics thanks to Google Analytics.
+* Infinite Scroll — Pulls the next posts automatically into view when the reader approaches the bottom of the page.
+* JSON API — Authorizes applications and services to securely connect to your blog, and allows them to use your content or offer you new functionality.
+* Lazy Load Images — Makes pages load faster by only lazy loading images that are on the screen, and loads other images as the user scrolls
+* Likes — Allows readers to show their appreciation for your posts with a single click.
+* Markdown — Allows you to compose posts and comments with links, lists, and other styles using regular characters and punctuation marks. Markdown is used by writers and bloggers who want a quick and easy way to write rich text without having to take their hands off the keyboard.
+* Notifications — Receive notifications for new comments and Likes in your admin bar and on your mobile device.
+* oEmbed Support — easily embed images, posts, and links from Facebook and Instagram.
+* Pinterest Block — Allows you to embed boards, profiles, and pins to your site.
```
Comment (jeherve) [inline readme.txt:150]: Why do we have this extra block here?

File: readme.txt [hunk 3]
```diff
@@ -1,97 +1,245 @@
...
+* Plugin Management — Allows easy site maintenance by choosing which plugins update automatically.
+* Post by Email — Publish posts using any mail client.
+* Protect — Protect your site from traditional and distributed brute force login attacks.
+* Publicize — Share new posts on social media networks automatically, or schedule future shares with custom messages.
+* Recurring Payments — Add a payment button specifically for a recurring donation, using Stripe as the payment gateway.
+* Related Posts  — Show contextual posts your visitors might be interested in reading after they’re done with their current post.
+* Secure Auth —  Secure WordPress.com powered login used by millions of sites with optional 2FA (two factor authentication) for extra protection.
```
Comment (jeherve) [inline readme.txt:157]: Is this the same as SSO, featured a few lines below?

File: readme.txt [hunk 4]
```diff
@@ -1,97 +1,245 @@
...
+* Likes — Allows readers to show their appreciation for your posts with a single click.
+* Markdown — Allows you to compose posts and comments with links, lists, and other styles using regular characters and punctuation marks. Markdown is used by writers and bloggers who want a quick and easy way to write rich text without having to take their hands off the keyboard.
+* Notifications — Receive notifications for new comments and Likes in your admin bar and on your mobile device.
+* oEmbed Support — easily embed images, posts, and links from Facebook and Instagram.
+* Pinterest Block — Allows you to embed boards, profiles, and pins to your site.
+* Plugin Management — Allows easy site maintenance by choosing which plugins update automatically.
+* Post by Email — Publish posts using any mail client.
+* Protect — Protect your site from traditional and distributed brute force login attacks.
+* Publicize — Share new posts on social media networks automatically, or schedule future shares with custom messages.
+* Recurring Payments — Add a payment button specifically for a recurring donation, using Stripe as the payment gateway.
```
Comment (jeherve) [inline readme.txt:155]: I think we've renamed that one into just Payments. Since it's a block, should we keep it in the below-list only?

**Review by jeherve — CHANGES_REQUESTED** [review]
I notice you've used `WP` and `Woo` in the plugin title, description, and tags. Is this shorthand appropriate for our users?

File: readme.txt (hunk 2 above)
Comment (robertbpugh) [reply readme.txt:150]: Oversight. I will delete.

File: readme.txt (hunk 3 above)
Comment (robertbpugh) [reply readme.txt:157]: Yes, I'll delete SSO

File: readme.txt (hunk 4 above)
Comment (robertbpugh) [reply readme.txt:155]: Yes

File: readme.txt [hunk 5]
```diff
@@ -1,97 +1,245 @@
...
-Tags: Jetpack, WordPress.com, backup, security, related posts, CDN, speed, anti-spam, social sharing, SEO, video, stats
+=== Jetpack - WP Security, Backup, Speed, Design, & Growth ===
```
[the remark this answers is not in the document]
Comment (robertbpugh) [reply readme.txt:1]: The top image is awkward. Will remove "Design,"

File: readme.txt [hunk 6]
```diff
@@ -1,97 +1,245 @@
...
+* See every site change and who made it with the activity log, great for coordination, debug, maintenance, or troubleshooting.
+* Automatic scan for malware and other code threats. One click fix to restore your site for malware.
+* Block spam comments and form responses with anti spam features powered by Akismet.
+* Brute force attack protection to protect your WordPress login page from attacks.
+* Monitor your site uptime / downtime and get an instant alert of any change by email.
+* Secure WordPress.com powered login used by millions of sites with optional 2FA (two factor authentication) for extra protection.
+* Auto update individual plugins for easy site maintenance and management.
+
+= PEAK SPEED AND PERFORMANCE =
+Get blazing fast site speed with Jetpack, the premier WP plugin built to leverage the power of AMP, a tool that helps optimize your site on mobile devices. Jetpack’s free CDN (content delivery network) auto optimizes your images and video. Watch your page load times decrease — we’ll optimize your images and serve them from our own powerful global network, and speed up your site on mobile devices to reduce bandwidth usage and save money!
```
[the remark this answers is not in the document]
Comment (robertbpugh) [reply readme.txt:31]: Yes, good catch.

File: readme.txt [hunk 7]
```diff
@@ -1,97 +1,245 @@
...
+
+* Google and Jetpack have partnered to create the best, highest performance all-in-one toolkit for WordPress. By using Jetpack and AMP together, you get all the features you need to build a beautiful, fast, modern website with no coding required.
```
[the remark this answers is not in the document]
Comment (robertbpugh) [reply readme.txt:33]: Ok will rephrase

File: readme.txt [hunk 8]
```diff
@@ -1,97 +1,245 @@
...
+* Lazy load images for a super fast experience, even on mobile. Jetpack’s lazy loading automatically delays the loading of media on your posts and pages until your visitors scroll down to where they appear on the page. 
+* Unlimited, high speed, ad free video hosting keeps the focus on your content, not on ads or recommendations that lead people off site.
+* Custom site search is incredibly powerful and customizable. Helps your visitors instantly find the right content so they read and buy more. Works great with WooCommerce / eCommerce sites to help filter products so customers get what they want on your site faster.
+* Recommended to use with WP Super Cache for ultimate WordPress site speed.
+
+= POWERFUL TOOLS FOR GROWTH =
+Create and customize your WordPress site, optimize it for visitors and revenue, and enjoy watching your stats tick up. Built it, share it, and watch it grow.
+
+* Advanced site stats and analytics to help you understand your audience.
+* Auto publish blog posts and products to social media by simply using our tools to connect to Facebook, Instagram, Twitter, and Linkedin.
```
[the remark this answers is not in the document]
Comment (robertbpugh) [reply readme.txt:44]: Will update

File: readme.txt [hunk 9]
```diff
@@ -1,97 +1,245 @@
...
+* Collect a payment or donation, sell a product, service, or membership with simple integrations with PayPal and Stripe.
+* SEO tools to grow traffic for Google, Bing, Twitter, Facebook, and WordPress.com to maximize your reach. XML sitemap created automatically.
```
[the remark this answers is not in the document]
Comment (robertbpugh) [reply readme.txt:46]: It's awkward, will rephrase

File: readme.txt [hunk 10]
```diff
@@ -1,97 +1,245 @@
...
+* Advertise on your site to generate revenue. The Jetpack ad network automatically does the work for you to find high-quality ads that are placed on your site.
+* Manage your site from anywhere with the official WordPress mobile app, available for Apple iOS (iPhone or iPad) and Google Android.
```
[the remark this answers is not in the document]
Comment (robertbpugh) [reply readme.txt:48]: Will make it more clear

File: readme.txt [hunk 11]
```diff
@@ -1,97 +1,245 @@
...
+Jetpack is updated monthly to ensure seamless integration with top WordPress plugins and other tech products.
+
+* Built for WooCommerce: Jetpack and WooCommerce are both made by Automattic. Backup, Scan, Anti-spam, integrate perfectly for Woo / eComm stores
+* Better understand your customers and marketing with Google Analytics (GA) integration
+* Social media platforms: Instagram, Facebook, Twitter, LinkedIn
+* Simple Blocks to customize your site: Pinterest, Whatsapp, Podcast player, GIFs, maps, tiled gallery, slideshow
+* Payment processors: easily collect payments or donations and sell products through Stripe and PayPal
+* Site speed and performance plugins: Works great with WP Super Cache and  Cloudflare.
+* Contact form: Anti-spam (Powered by Akismet) blocks spam comments for Jetpack forms and Contact Form 7.
+* Other tech integrations: Creative Mail, Mailchimp, Calendly, Whatsapp, Pinterest
```
[the remark this answers is not in the document]
Comment (robertbpugh) [reply readme.txt:72]: Yes, will update

File: readme.txt [hunk 12]
```diff
@@ -1,97 +1,245 @@
...
+ 
+Jetpack has an image CDN that works by caching and serving your WordPress images globally from its own servers. These plugins are both maintained by Automattic and work together to give you ultimate site speed.
+
+= What version of PHP do I need? =
+
+Sites must be built on PHP 5.6 or greater, but Jetpack always supports the latest version of PHP.
+ 
+= Can Jetpack help my site comply with GDPR? =
+
+You can easily comply with GDPR with our Cookie and Consent Banner. The European Union’s ePrivacy Directive (often referred to as the ‘cookie law’) and General Data Protection Regulation (GDPR) place requirements on website owners and operators to provide information about, and gain consent for their use of cookies.
```
[the remark this answers is not in the document]
Comment (robertbpugh) [reply readme.txt:237]: Updated

**Review by jeherve — APPROVED** [review]

**Comment by jeherve** [comment]
Cherry-picked to `branch-9.0` in 880750f543ad2fb90ddb77e2dae277de83b2d335

**Pull request merged**